suggester/src/main/java/org/opengrok/suggest/Suggester.java

```diff
@@ -234,6 +249,11 @@ public void rebuild(final Collection<NamedIndexDir> indexDirs) {
             return;
         }
 
+        // for testing
```

Comment:
Is it needed?

Reply:
you mean the comment ?

Reply:
Yes, it is in the PR at least 3 times. This kind of comment indicates that the code should be removed in production code.